cmd/cli/user.go

```diff
@@ -108,11 +108,19 @@ func usersCleanup(logger moira.Logger, database moira.Database, users []string,
 		usersMap[user] = true
 	}
 
+	logger.Info().
+		Int("count", len(usersMap)).
+		Msg("actual users")
```

Comment:
I think we use capital leters, since it's a complete sentence
```suggestion
		Msg("Actual users")
```